### PR TITLE
Performance issues with metadatas

### PR DESCRIPTION
**Pull request by dmitryuk**
Closes https://github.com/DamienHarper/auditor-bundle/issues/319
Metadata cache should be configured to start it working

### CODE REVIEW

File: src/Provider/Doctrine/DoctrineProvider.php [hunk 1]
```diff
@@ -238,14 +239,24 @@ public function setStorageMapper(callable $storageMapper): void
         $this->configuration->setStorageMapper($storageMapper);
     }
 
-    public function loadAnnotations(EntityManagerInterface $entityManager, ?array $entities = null): self
```
Comment (dmitryuk) [inline src/Provider/Doctrine/DoctrineProvider.php:241]: `$entities=null` never passed

File: src/Provider/Doctrine/Auditing/Transaction/TransactionHydrator.php [hunk 2]
```diff
@@ -103,7 +103,7 @@ private function hydrateWithScheduledCollectionUpdates(Transaction $transaction,
 
                 /** @var object $entity */
                 foreach ($collection->getDeleteDiff() as $entity) {
-                    if ($this->provider->isAudited($entity) && $collection->getOwner()) {
+                    if ($this->provider->isAudited($entity)) {
```
Comment (dmitryuk) [inline src/Provider/Doctrine/Auditing/Transaction/TransactionHydrator.php:106]: where is already this check on line 86

**Review by DamienHarper — APPROVED** [review]

**Comment by DamienHarper** [comment]
Thanks again @dmitryuk! This one was on my roadmap :)
Do you have some sort of before/after benchmark to better understand the gain?

**Comment by dmitryuk** [comment]
Test example:

```
    public function testExampleAsd()
    {
        $container = self::getContainer();
        $provider = new DoctrineProvider(new Configuration([]));
        $em = $container->get('doctrine.orm.entity_manager');

        for ($i = 0; $i< 50; $i++) {
            $provider->loadAnnotations($em, []);
        }
    }
```
Without cache: `Time: 00:26.980, Memory: 52.50 MB`

With cache: `Time: 00:01.702, Memory: 46.50 MB`

**Pull request merged**